packages/core/README.md

```diff
@@ -24,6 +24,7 @@ The agent requires a configuration parameter with the following properties:
 | Property                | Description                                                             | Type                  | Default Value Variable                                   |
 | ----------------------- | ----------------------------------------------------------------------- | --------------------- | -------------------------------------------------------- |
 | `app`                   | Application metadata                                                    | `App`                 |                                                          |
+| `dedupe`                | A flag for toggling deduplication filter                                | `boolean`             |                                                          |
```

Comment:
Should we call out the default value here?

Reply:
In the core package there's no default :-/. Although `undefined == false`, one can't initialize the core package without manually providing this flag.